yksgi/src/llvmbridge.rs

```diff
@@ -40,116 +42,140 @@ impl LLVMFunction {
         Self(func)
     }
 
-    pub unsafe fn bb(&self, bbidx: usize) -> LLVMBasicBlock {
-        let mut bb = LLVMGetFirstBasicBlock(self.0);
-        for _ in 0..bbidx {
-            bb = LLVMGetNextBasicBlock(bb);
+    pub fn bb(&self, bbidx: usize) -> LLVMBasicBlock {
+        unsafe {
```

Comment:
Similarly here, the scope of `unsafe` can be constrained.

Reply:
Fixed in 9a253a4